Assets/Scripts/NetworkManager.cs

```diff
@@ -27,7 +27,9 @@ void Start()
                 Instance = this;
                 Object.DontDestroyOnLoad(gameObject);
                 PhotonNetwork.automaticallySyncScene = true;
-                if (!SceneManager.GetSceneByName("StartMenu").isLoaded)

```

Comment:
I can no longer start from the Main scene or the ReadyRoom scene in the Unity Editor.